changelog.mdx

```diff
@@ -5,6 +5,32 @@ metaDescription: Chalk's latest updates.
 description: Updates to Chalk!
 ---
 
+## January 6, 2025
+
+## Reduced cloud costs from optimized resource usage with KEDA
+
+We’re excited to share that we’ve implemented **KEDA (Kubernetes-based Event Driven Autoscaler)** and are rolling it out over the next few weeks.
+With KEDA, your environment now dynamically scales up and down—even to **zero machines** when there’s reduced or no workload.
+With automatic scaling we're able to optimize resource usage and reduce cloud costs--expect to see a drop in your cloud spend as a result.
+
+## View logs using the CLI
+
+View logs using the Chalk CLI! Inspect your log stream the same way you would from the GUI but via the command line!
```

Comment:
i'd be hesitant to use exclamation points, and would definitely shy away from using two twice in a row